Tasks/PowerShellV2/powershell.ts

```diff
@@ -67,7 +67,7 @@ async function run() {
         let tempDirectory = tl.getVariable('agent.tempDirectory');
         tl.checkPath(tempDirectory, `${tempDirectory} (agent.tempDirectory)`);
         let filePath = path.join(tempDirectory, uuidV4() + '.ps1');
-        await fs.writeFile(
+        fs.writeFileSync(
```

Comment:
Did we end to end test that task to make sure this code executes with no issues?